matching-service/src/main/java/de/muenchen/oss/swim/matching/adapter/out/csv/CsvAdapter.java

```diff
@@ -0,0 +1,59 @@
+package de.muenchen.oss.swim.matching.adapter.out.csv;
+
+import com.fasterxml.jackson.databind.MappingIterator;
+import com.fasterxml.jackson.dataformat.csv.CsvMapper;
+import com.fasterxml.jackson.dataformat.csv.CsvSchema;
+import de.muenchen.oss.swim.matching.application.port.out.ExportParsingOutPort;
+import de.muenchen.oss.swim.matching.domain.exception.CsvParsingException;
+import de.muenchen.oss.swim.matching.domain.model.DmsInbox;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class CsvAdapter implements ExportParsingOutPort {
+    private static final char CSV_DELIMITER = ';';
+
+    private final DmsExportMapper dmsExportMapper;
+
+    @Override
+    public List<DmsInbox> parseCsv(final InputStream exportContent) throws CsvParsingException {
+        final CsvMapper csvMapper = new CsvMapper();
+        final CsvSchema schema = csvMapper.typedSchemaFor(DmsInboxDTO.class)
+                .withHeader()
+                .withColumnSeparator(CSV_DELIMITER)
+                .withColumnReordering(true);
+        try (MappingIterator<DmsInboxDTO> iterator = csvMapper
+                .readerFor(DmsInboxDTO.class)
+                .with(schema)
+                .readValues(exportContent)) {
+            final List<DmsInboxDTO> dtos = new ArrayList<>();
+            int lines = 0;
+            int skippedLines = 0;
+            while (iterator.hasNextValue()) {
+                lines++;
+                try {
+                    dtos.add(iterator.nextValue());
+                } catch (final IOException e) {
+                    skippedLines++;
+                    log.debug("Failed to parse line {}: {}", lines, e.getMessage());
```

Comment:
I would like to have this set to `info`. I understand that this will initially bloat the logfile, but should help in increasing the data quality in the productive eAkte system.

Reply:
Will be configured in deployment with:
```
logging:
  level:
    de.muenchen.oss.swim.matching.adapter.out.csv.CsvAdapter: debug
```